Form/Type/BooleanType.php

```diff
@@ -81,7 +81,9 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getParent()
     {
-        return 'choice';
+        return class_exists('Symfony\Component\Form\Extension\Core\Type\ChoiceType') ?
```

Comment:
Maybe this is a logical inversion problem? Can you copy / paste your code here ?